samples/test/util/configUtils.js

```diff
@@ -29,6 +29,9 @@ function getConfig() {
   const password = process.env.PASSWORD;
   const webClientId = process.env.WEB_CLIENT_ID || process.env.CLIENT_ID;
   const clientSecret = process.env.WEB_CLIENT_SECRET || process.env.CLIENT_SECRET;
+  const a18nAPIKey = process.env.A18N_API_KEY;
```

Comment:
these values need to be stored securely for access by Bacon. (We may also need these values for local testing). @vijetmahabaleshwar-okta can you help?